examples/sample-app/README.md

```diff
@@ -281,6 +308,15 @@ the present working directory is the same directory as this README.
 
 19. Confirm the application is now accessible via the frontend service on port 5432.  Go to http://172.30.17.4:5432 (or whatever IP address was reported above) in your browser if you're running this locally; otherwise you can use curl to see the HTML, or port forward the address to your local workstation to visit it.
 
+	- - -
+	**VAGRANT USERS:**
+	Open a new terminal and follow these two commands from the origin directory to fire up a vagrant instance and port forward the address to your local worksation.
```

Comment:
* the last L in terminaL should be lowercase.
* `vagrant up` is not necessary since Vagrant is already running

I recommend changing this to "Open a new terminal and enter this command to forward the application port to a port on your workstation:"